base/dft.jl

```diff
@@ -580,7 +580,7 @@ module FFTW
     """
     function plan_r2r end
 
-    Base.USE_GPL_LIBS && include(joinpath("fft", "FFTW.jl"))
+    (Base.USE_GPL_LIBS || Base.USE_INTEL_MKL_FFT) && include(joinpath("fft", "FFTW.jl"))
```

Comment:
Instead of using the constants from the Makefile, you can change this check to: 

`(Base.USE_GPL_LIBS || Base.fftw_vendor() == :mkl)`

You don't need the constants in the Makefile thereafter.